lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
I updated that page to use #{WorkshopConstants::PROGRAM_TYPES.to_json}; 
and verified it in the browser.